AsyncDisplayKit/ASControlNode+Subclasses.h

```diff
@@ -20,6 +20,12 @@ NS_ASSUME_NONNULL_BEGIN
 @interface ASControlNode (Subclassing)
 
 /**
+ @abstract Indicates whether or not at least one target was added to the receiver
+ @discussion YES if the receiver has at least one target; NO otherwise.
+ */
+@property (nonatomic, readonly, assign, getter=isTargetAdded) BOOL targetAdded;
```

Comment:
@appleguy Yep makes sense. The initial thoughts was that e.g. ASTextNode would be able to check if a target was added, but you are right there is no use case at the moment. I removed the property and moved to use the instance variable only.